bundles/admin/admin-layereditor/components/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'antd/lib/button';
+import 'antd/lib/button/style/css';
+
+export const ButtonComponent = (props) => {
+    const {text, ...other} = props;
+    return (
+        <Button {...other}>{text}</Button>
```

Comment:
We could pass in `props.children` into the Button instead of `text` to allow more customizable buttons, not just text-buttons. https://reactjs.org/docs/composition-vs-inheritance.html#containment